src/components/list/list.css

```diff
@@ -0,0 +1,13 @@
+.artsy-list{
```

Comment:
in global css you can write these code ==> to avoid repetaion because we rarely need the html default property

ul {list-style : none ; }
a {text-decoration: none; }